### ISSUE TITLE
Unable to install and activesupport error

### ISSUE THREAD

**Issue by oshovah** [refinery/refinerycms-page-images#121]
Hi

Rails 4.1.5
Ruby 2.1.2
Refinerycms-edge

Unfortunately its impossible to install the latest page-images.
```
gem 'refinerycms-page-images', '~> 3.0.0'
```
```
Could not find gem 'refinerycms-page-images (~> 3.0.0) ruby' in the gems available on this machine.
```
I change the Gemfile
```
gem 'refinerycms-page-images', branch: 'master'
```
but after ```rails generate refinery:page_images``` i get this error:
```
$ rails generate refinery:page_images
/Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/core_ext/hash/keys.rb:71:in `block in assert_valid_keys': Unknown key: :order. Valid keys are: :class_name, :class, :foreign_key, :validate, :autosave, :table_name, :before_add, :after_add, :before_remove, :after_remove, :extend, :primary_key, :dependent, :as, :through, :source, :source_type, :inverse_of, :counter_cache, :join_table (ArgumentError)
```
Long Version:
```
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/core_ext/hash/keys.rb:69:in `each_key'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/core_ext/hash/keys.rb:69:in `assert_valid_keys'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activerecord-4.1.5/lib/active_record/associations/builder/association.rb:81:in `validate_options'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activerecord-4.1.5/lib/active_record/associations/builder/association.rb:61:in `initialize'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activerecord-4.1.5/lib/active_record/associations/builder/collection_association.rb:18:in `initialize'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activerecord-4.1.5/lib/active_record/associations/builder/association.rb:46:in `new'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activerecord-4.1.5/lib/active_record/associations/builder/association.rb:46:in `create_builder'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activerecord-4.1.5/lib/active_record/associations/builder/association.rb:35:in `build'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activerecord-4.1.5/lib/active_record/associations.rb:1215:in `has_many'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/refinerycms-page-images-1.0.4/lib/refinerycms-page-images.rb:13:in `block (2 levels) in <class:Engine>'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/refinerycms-page-images-1.0.4/lib/refinerycms-page-images.rb:11:in `module_eval'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/refinerycms-page-images-1.0.4/lib/refinerycms-page-images.rb:11:in `block in <class:Engine>'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:438:in `instance_exec'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:438:in `block in make_lambda'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:184:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:184:in `block in simple'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `block in simple'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `block in simple'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `block in simple'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `block in simple'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `block in simple'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `block in simple'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:185:in `block in simple'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:86:in `call'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/callbacks.rb:86:in `run_callbacks'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/actionpack-4.1.5/lib/action_dispatch/middleware/reloader.rb:83:in `prepare!'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/actionpack-4.1.5/lib/action_dispatch/middleware/reloader.rb:55:in `prepare!'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/application/finisher.rb:52:in `block in <module:Finisher>'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:30:in `instance_exec'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:30:in `run'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:55:in `block in run_initializers'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:226:in `block in tsort_each'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:348:in `block (2 levels) in each_strongly_connected_component'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:418:in `block (2 levels) in each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:418:in `block (2 levels) in each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:418:in `block (2 levels) in each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:418:in `block (2 levels) in each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:427:in `each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:417:in `block in each_strongly_connected_component_from'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:44:in `each'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:44:in `tsort_each_child'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:411:in `call'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:411:in `each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:417:in `block in each_strongly_connected_component_from'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:44:in `each'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:44:in `tsort_each_child'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:411:in `call'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:411:in `each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:417:in `block in each_strongly_connected_component_from'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:44:in `each'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:44:in `tsort_each_child'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:411:in `call'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:411:in `each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:417:in `block in each_strongly_connected_component_from'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:44:in `each'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:44:in `tsort_each_child'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:411:in `call'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:411:in `each_strongly_connected_component_from'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:347:in `block in each_strongly_connected_component'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:345:in `each'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:345:in `call'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:345:in `each_strongly_connected_component'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:224:in `tsort_each'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/2.1.0/tsort.rb:205:in `tsort_each'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/initializable.rb:54:in `run_initializers'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/railties-4.1.5/lib/rails/application.rb:300:in `initialize!'
	from /Users/gott/htdocs/_refinerycms-edge/config/environment.rb:5:in `<top (required)>'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/dependencies.rb:247:in `require'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/dependencies.rb:247:in `block in require'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/dependencies.rb:232:in `load_dependency'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/activesupport-4.1.5/lib/active_support/dependencies.rb:247:in `require'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/spring-1.2.0/lib/spring/application.rb:92:in `preload'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/spring-1.2.0/lib/spring/application.rb:143:in `serve'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/spring-1.2.0/lib/spring/application.rb:131:in `block in run'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/spring-1.2.0/lib/spring/application.rb:125:in `loop'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/spring-1.2.0/lib/spring/application.rb:125:in `run'
	from /Users/gott/htdocs/_refinerycms-edge/.gems/ruby/2.1.0/gems/spring-1.2.0/lib/spring/application/boot.rb:18:in `<top (required)>'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/gott/.rbenv/versions/2.1.2/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
```

**Comment by parndt** [comment]
Try:

```ruby
gem "refinerycms-page-images", github: "refinery/refinerycms-page-images", branch: "master"
```

**Comment by oshovah** [comment]
Thanks! It Works with this

**Comment by adolfopeccin** [comment]
Using gem "refinerycms-page-images", github: "refinery/refinerycms-page-images", branch: "master" is causing problem with globalize right now.

**Comment by Iraneus** [comment]
@parndt  Nice work!
Could your recommendations be updated here https://github.com/refinery/refinerycms-page-images
Cheers

**Comment by parndt** [comment]
@Iraneus my sole recommendation was to try the latest master branch of this project 😄

**Comment by parndt** [comment]
Closed by 4f40c58c492a6eed9a876f10c2c1e06402d294f9
